tests/GetLicenseDataTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * How to set up and run tests:
+ *
+ * 1. Install PHPUnit via Composer:
+ *    composer require --dev phpunit/phpunit
```

Comment:
You already have installed with `composer.json`. So one just needs to do `composer install`.